src/Docfx.Dotnet/MetadataJsonConfig.cs

```diff
@@ -258,6 +258,13 @@ internal class MetadataJsonItemConfig
     [JsonProperty("allowCompilationErrors")]
     [JsonPropertyName("allowCompilationErrors")]
     public bool AllowCompilationErrors { get; set; }
+
+    /// <summary>
+    ///   When enabled, the types uses the CLR type names instead of the C# aliases.
+    /// </summary>
+    [JsonProperty("useClrTypeNames")]
+    [JsonPropertyName("useClrTypeNames")]
+    public bool UseClrTypeNames { get; init; }
```

Comment:
Hey @Patrick8639, this PR looks great! I want to check if this change is included in the PR by mistake, or do you intend to implement the feature in another PR?

Reply:
Hey @yufeih It was an error when creating the pull request...
I really don't know how I have pushed the same code in two push requests!
The change is for https://github.com/dotnet/docfx/pull/10072 and I will try to remove from this one.

Reply:
No problem at all, thank you for making a contribution!